modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -81,9 +86,15 @@ object FormInputEV {
   type ChangeCallback[A] = A => Callback
 
   @Lenses
-  final case class State(curValue: String, prevValue: String, errors: Option[NonEmptyChain[String]])
+  final case class State(
+    displayValue: String,
+    modelValue:   String,
+    cursor:       Option[(Int, Int)],
+    errors:       Option[NonEmptyChain[String]]
+  )
 
   class Backend[EV[_], A]($ : BackendScope[Props[EV, A], State]) {
+    private val outerRef = Ref[html.Element]
```

Comment:
Is this a reff to the `input`?

Reply:
It's a Ref to the new `div` I had to wrap around the `FormInput`. There is no way to get a ref directly to the `input`. It also isn't possible to get a Ref to the FormInput itself because it is a function component.